ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -599,6 +599,12 @@ func (r *RayClusterReconciler) reconcilePods(ctx context.Context, instance *rayv
 		}
 	}
 
+	// if RayCluster is suspended, delete all pods and skip reconcile
+	if instance.Spec.Suspend != nil && *instance.Spec.Suspend {
+		filterLabels = client.MatchingLabels{common.RayClusterLabelKey: instance.Name}
```

Comment:
Should the cluster state be checked and the deletion skipped when it has already been suspended?

Reply:
done